### DISCUSSION TITLE
Amplify as part of the SEA

### DISCUSSION THREAD

**Discussion by Bhawllar** [aws-samples/aws-secure-environment-accelerator#605] [Q&A] [answered]
The government of Canada will end up deploying web applications on AWS. One of the ways to ensure that there is reduced security risk is to avoid the use of webforms and engines that can create security nightmare. If Amplify and Cognito are part of the SEA deployment, it will help with marketing to government clients. Is Amplify and Cognito part of SEA 1.2.5 or what is the best way to ensure that these services are added to 1.2.4

Thanks!

**Comment by jodlynch** [comment, marked as answer]
The Secure Environment Accelerator is meant to build a secure foundational AWS environment that aligns with best-practice security architectures and specific compliance frameworks (e.g. Government of Canada Protected B Medium Medium).  This is often referred to as a "Landing Zone" in the generic sense.  It's not meant to be an accelerator for every AWS Service or every workload that a customer may want to implement. In fact, the intent is to only setup the foundational environment that puts the necessary structure, controls, and services in place to allow customers to then add workloads at scale that can leverage this foundation.  Services like Amplify and Cognito can be used within the context of this approach as customers choose to do so without the need for any additions to the Accelerator.  Customers are free to implement these services how they want, develop automation of their own for these services, and deploy them within the account structure setup by the Accelerator.  There are many choices for customers when it comes to building workloads/applications and the Accelerator aims not to limit that choice or prescribe which services should be used.
